### DISCUSSION TITLE
Is Apisix Dashboard not actively maintained?

### DISCUSSION THREAD

**Discussion by MirtoBusico** [apache/apisix#9871] [Q&A] [answered]
Hi all,
what is the destiny of Apisix Dashboard?

If it will be dismissed, anyone plans to develop a graphic tool to manage Apisix?

**Comment by bzp2010** [comment]
We are planning to refactor the project and limit its scope to make it maintainable.

ref: apache/apisix-dashboard#2981

**Comment by SkyeYoung** [comment, marked as answer]
Hi, we have built a new dashboard in APISIX 3.13.0. The old version is no longer maintained. You can refer to:

1. [notice (release)](https://github.com/apache/apisix-dashboard/releases/tag/notice)
2. Apache APISIX Dashboard Roadmap & Scopes #2981
3. [apisix.apache.org/docs/apisix/next/getting-started/README#validate](https://apisix.apache.org/docs/apisix/next/getting-started/README/#validate)
